synchronization/pipeline.cc

```diff
@@ -0,0 +1,11 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/synchronization/pipeline.h"
+
+namespace flutter {
+
+//
```

Comment:
If this file just exists to check the build for the header, I usually skip the namespace and write a comment to that effect.